src/publication.py

```diff
@@ -52,4 +56,5 @@ def serialize(self):
             "rssURL": self.publication["rssURL"],
             "slug": self.publication["slug"],
             "websiteURL": self.publication["websiteURL"],
+            "contentTypes": self.publication["contentTypes"],
```

Comment:
i see where the contentTypes comes from - make sure this is in the PR description